edi_oca/models/edi_exchange_type.py

```diff
@@ -151,6 +151,39 @@ class EDIExchangeType(models.Model):
         "you should define a sequence in this field to show "
         "the sequence in your filename",
     )
+    # https://docs.python.org/3/library/codecs.html#standard-encodings
+    encoding_out = fields.Char(
```

Comment:
Why do you need out and in? An edi.exchange.type can only be input or output, but not both :thinking: 

With one might be enough, isn't it?

Reply:
@etobella true, for the encoding I can keep one field, although for the error handler there is one option that is only available for encoding, so maybe we can keep it separated and only show the relevant field depending on the direction.

Reply:
Ok, makes sense to me